src/Farmhand.js

```diff
@@ -390,6 +396,12 @@ export default class Farmhand extends Component {
       todaysPurchases: {},
       todaysRevenue: 0,
       todaysStartingInventory: {},
+      toolLevels: {
+        [toolType.HOE]: toolLevel.DEFAULT,
+        [toolType.SCYTHE]: toolLevel.DEFAULT,
+        [toolType.SHOVEL]: null,
```

Comment:
yeah, i went back and forth a bit but landed on this so that things checking `toolLevels` can assume there's an entry for every _possible_ tool, but the `null` means it isn't unlocked yet. i suppose a `toolLevel.LOCKED` could exist? the "unlock" code though is what updates the shovel to the default level

Reply:
Ahhh that makes sense! In that case I think this would benefit from either a comment explaining the special case value or using a new enum value like `toolLevel.LOCKED` (though I'd prefer something a little less ambiguous like `toolLevel.UNAVAILABLE`). The second option doesn't seem like it requires explanation via comments, so that's perhaps the better route to take here. 🙂

Reply:
just made the update to add `toolType.UNAVAILABLE` and it reads nicely!